core/src/test/java/apoc/trigger/TriggerClusterRoutingTest.java

```diff
@@ -31,7 +31,7 @@ public class TriggerClusterRoutingTest {
     public static void setupCluster() {
         cluster = TestContainerUtil
                 .createEnterpriseCluster( List.of(TestContainerUtil.ApocPackage.CORE), 
-                        3, 1, 
+                        3, 0,
```

Comment:
Don't allocate a read replica. We weren't testing anything with the read replica anyway (because we were skipping it inside the tests) and read replicas have a different behaviour in 5.x anyway, so it was failing to start up